laurielim/Druid-React22k-demo
# AW-27 basic news config

Related ticket https://druidfi.atlassian.net/browse/AW-27

In this PR:
- created new content type news
- configured form display and display

To test:
- run `make fresh `
- run `drush cim drush updb drush cr`
- go to admin page
- create new content type news with text field
- see that content is rendered (header hides content, check source)
